core/src/test/java/org/elasticsearch/search/aggregations/bucket/adjacency/InternalAdjacencyMatrixTests.java

```diff
@@ -37,22 +39,24 @@
     @Override
     public void setUp() throws Exception {
         super.setUp();
-        keys = new ArrayList<>();
+        List<String> listOfKeys = new ArrayList<>();
         int numFilters = randomIntBetween(2, 4);
```

Comment:
I agree, I wasn't fully comfortable with that. What you suggest makes sense, I updated the code.